drivers/interrupt_controller/exti_stm32.c

```diff
@@ -625,6 +466,9 @@ static void __stm32_exti_connect_irqs(struct device *dev)
 		CONFIG_EXTI_STM32_EXTI15_10_IRQ_PRI,
 		__stm32_exti_isr_15_10, DEVICE_GET(exti_stm32),
 		0);
+#elif defined (ONFIG_SOC_SERIES_STM32F2X) || \
```

Comment:
CONFIG?

Reply:
Oups, forgot to amend before pushing... Fixed, thanks!